queries.sql

```diff
@@ -7,4 +7,51 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 and 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 and 17.3;
+
+-- Transaction 1
+BEGIN;
+UPDATE ANIMALS SET species='unspecified';
+ROLLBACK;
+SELECT * FROM animals;
+
+-- Transaction 2
+BEGIN;
+UPDATE ANIMALS SET species='digimon' WHERE name like '%mon';
+UPDATE ANIMALS SET species='pokemon' WHERE species is null;
+COMMIT;
+SELECT * FROM animals;
+
+-- Transaction3
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+-- Transaction 4
+BEGIN;
+DELETE FROM animals where date_of_birth>'2022-01-01';
+SAVEPOINT point1;
+UPDATE animals SET weight_kg=weight_kg*-1;
+ROLLBACK TO point1;
+UPDATE animals SET weight_kg=weight_kg*-1 WHERE weight_kg<0;
+COMMIT;
+SELECT * FROM animals ORDER BY id;
+
+/*More Queries.*/
+SELECT COUNT(name) AS total_animals FROM animals;
+
+SELECT COUNT(escape_attempts) AS animals_tried_escaping
+  FROM animals WHERE escape_attempts>0;
```

Comment:
- [ ] The question for this query asks for animals that have never tried to escape which means the `escape_attempts` should be equal to not greater than 0.